
Don't disable dlopen just because of cross-compiling, if it's available it most likely works

--- configure
+++ configure
@@ -17682,7 +17682,7 @@
     ac_libs=$LIBS
     LIBS="$LIBS -ldl"
     if test "$cross_compiling" = yes; then :
-  found=no
+# assume if it's available it most likely works on the target
 else
   cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
